src/OrbitGl/TimelineUiTest.cpp

```diff
@@ -53,17 +53,24 @@ class TimelineUiTest : public TimelineUi {
     }
     EXPECT_LE(num_major_ticks, 10);
 
-    // Depending on the scale, there should be 1 minor ticks or 4 between each major tick, which
+    // Depending on the scale, there should be 1, 2 or 4 minor ticks between each major tick, which
     // means that (calling MT to the number of major ticks, and mt to the number of minor ticks)
-    // mt is between (MT-1, MT+1) or mt is between (4x(MT-1), 4x(MT+1)).
+    // mt is between (MT-1, MT+1), (2*(MT-1), 2x(MT+1)) or (4x(MT-1), 4x(MT+1)).
```

Comment:
I prefer to use `*` everywhere for multiplication. It is inconsistent with `x` here